staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1beta1/register.go

```diff
@@ -44,6 +48,9 @@ func init() {
 
 // Adds the list of known types to the given scheme.
 func addKnownTypes(scheme *runtime.Scheme) error {
+	scheme.AddKnownTypes(LegacySchemeGroupVersion,
+		&EgressSelectorConfiguration{},
```

Comment:
all these legacy/new group name changes should not be mixed into the tracing commit.

Reply:
Moved to https://github.com/kubernetes/kubernetes/pull/99906